### PR TITLE
user form autosave

### PR DESCRIPTION
**Pull request by rfontanarosa**
part of #1520

- [x] implement auto save for all user properties
- [x] Remove buttons

### CODE REVIEW

File: src/client/components/EditUserForm/EditUserForm.tsx [hunk 1]
```diff
@@ -2,15 +2,13 @@ import './EditUserForm.scss'
...
+import { RoleName, User, UserRole, Users } from '@meta/user'
 
 import { useAppDispatch } from '@client/store'
 import { useAssessment, useCycle } from '@client/store/assessment'
 import { UserManagementActions } from '@client/store/userManagement'
-import { useCountryIso } from '@client/hooks'
-import { useToaster } from '@client/hooks/useToaster'
+import { useCountryIso, useOnUpdate } from '@client/hooks'
 
-import Buttons from './Buttons'
```
Comment (minotogna) [inline src/client/components/EditUserForm/EditUserForm.tsx:13]: if not used anymore, it should be removed

File: src/server/repository/public/user/readCountryUsersByRole.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { BaseProtocol, DB } from '@server/db'
   We get a list of recipients for Array<<countryIsos>> and Array<<RoleName>>
 */
 
-const fields: Array<string> = ['lang', 'id', 'name', 'status', 'position', 'email']
+const fields: Array<string> = ['lang', 'id', 'name', 'status', 'position', 'email', 'institution']
```
Comment (minotogna) [inline src/server/repository/public/user/readCountryUsersByRole.ts]: better to move this into a common file './fields' and use this in getOne and here

File: src/client/components/EditUserForm/EditUserForm.tsx [hunk 3]
```diff
@@ -19,72 +17,58 @@ import TextInputFields from './TextInputFields'
...
-      />
+      <div className="edit-user__form-item edit-user__form-item-buttons">
+        <div className="edit-user__form-label" />
+        <div className="edit-user__form-field edit-user__form-field-buttons">
+          <button
+            type="button"
+            className="btn btn-secondary"
+            onClick={() => dispatch(UserManagementActions.setUserToEdit(null))}
+          >
+            {i18n.t<string>('editUser.cancel')}
```
Comment (sorja) [inline src/client/components/EditUserForm/EditUserForm.tsx:68]: should start using `t` over `i18n.t` as `const { t } = useTranslation()`

**Review by sorja — APPROVED** [review]

**Review by minotogna — APPROVED** [review]

File: src/client/components/EditUserForm/EditUserForm.tsx (hunk 3 above)
Comment (rfontanarosa) [reply src/client/components/EditUserForm/EditUserForm.tsx:68]: ok

**Pull request merged**